glom/core.py

```diff
@@ -870,6 +879,8 @@ def _t_eval(target, _t, scope):
             try:
                 cur = get(cur, arg)
             except Exception as e:
+                if op == '?':
```

Comment:
Seems to me that this condition is a bit too broad. Better may be to check for a specific subset of exceptions. Better than that would be to only have this behavior if `cur` is itself `None` (assuming that's the only null-equivalent value we're interested in supporting). Does that seem right?